cluster_experiments/random_splitter.py

```diff
@@ -481,3 +481,56 @@ def from_config(cls, config) -> "StratifiedSwitchbackSplitter":
             splitter_weights=config.splitter_weights,
             washover=washover_cls.from_config(config),
         )
+
+
+class BacktestSplitter(RandomSplitter):
+    """
+    Doesn't actually split the data, but duplicates all rows for all treatments.
```

Comment:
in this case, wdyt about renaming to something more specific about the sampler? like RepeatedSampler or something like that